webpack.config.js

```diff
@@ -35,17 +37,40 @@ function getLoadersConfig(env) {
   }
 }
 
+function getAllPossibleLanguages() {
+  var srcDirectory = "app";
+  return fs.readdirSync(srcDirectory)
+    .filter(function(file) {
+      if(/^config\.\w+\.coffee$/.test(file)) return file;
+    })
+    .filter(function(file) {
+      if(!/^config\.client\.coffee$/.test(file)) return file;
+    })
+    .map(function(file) {
+      var config = require("./"+srcDirectory+"/"+file);
+      return config.availableLanguages;
+    })
+    .reduce(function(languages, languages2) {
+      return languages.concat(languages2);
+    })
+    .filter(function(language, position, languages) {
+      return languages.indexOf(language) == position;
+    });
+}
+
 function getPluginsConfig(env) {
+  var languageExpression = new RegExp("^./(" + getAllPossibleLanguages().join('|') + ")$");
+
   if (env === "development") {
     return([
       new webpack.HotModuleReplacementPlugin(),
-      new webpack.ContextReplacementPlugin(/moment(\/|\\)locale$/, /fi|sv|en\-gb/),
+      new webpack.ContextReplacementPlugin(/moment[\\\/]locale$/, languageExpression),
```

Comment:
Nice, now the same has to be done for `react-intl/lib/locale-data`, as it still tries to put evertyhing to the webpack context